scss/_patterns_pull-quotes.scss

```diff
@@ -65,7 +64,7 @@
   }
 
   .p-pull-quote {
-    padding: 0 $sph-outer * 2;
+    padding: 0 2rem;
```

Comment:
Is it intended that these values are hardcoded, rather than variables?

Reply:
These are offsets to fit a quote at the size it is used in each pattern, done by eyeballing what looks good basically - and this has nothing to do with keeping spacing consistent across components, as no other components use quotes at those font sizes. I just wanted that to be clear. 

For example, if you want to change that quote size, and respectively the spacing around it, you wouldn't expect anything outside the quote component to also change - therefore it shouldn't use $sph-outer.

And viceversa, if you changed `$sph-outer` so that components like the matrix gert a different spacing of the icon, you wouldn't expect the quote spacing to change as it is unrelated.

Reply:
If pull quote is a special case in terms of spacing we should comment on it in the code, so we know why vars are not used here.

Maybe worth adding some internal vars for this component only to be used instead?

Reply:
My goal initially was to decouple this from the vars as it is an incorrect coupling. Didn't use vars as it isn't used in more than one place. The correct way to do this is to measure the width of the qutaiton mark in ems, add a spacing var and use that. I'll add an issue for that.

Reply:
Issue created here https://github.com/canonical-web-and-design/vanilla-framework/issues/3821